concent_api/verifier/tasks.py

```diff
@@ -12,11 +24,49 @@
 @shared_task
 def blender_verification_order(
     subtask_id:             str,
-    source_package_path:    str,  # pylint: disable=unused-argument
-    result_package_path:    str,  # pylint: disable=unused-argument
+    source_package_path:    str,
+    result_package_path:    str,
     output_format:          Enum,  # pylint: disable=unused-argument
     scene_file:             str,  # pylint: disable=unused-argument
+    report_computed_task:   message.ReportComputedTask,
 ):
+    # Generate a FileTransferToken valid for a download of any file listed in the order.
+    file_transfer_tokens = {}
+
+    # !! Files MUST be added in following order - sources first, them results !!
+    for file_path in (source_package_path, result_package_path):
+        file_transfer_tokens[file_path] = create_file_transfer_token_for_concent(
+            report_computed_task=report_computed_task,
+            operation=message.FileTransferToken.Operation.download,
+            file_path=file_path,
+        )
+
+    # Remove any files from VERIFIER_STORAGE_PATH.
+    clean_directory(settings.VERIFIER_STORAGE_PATH)
+
+    # Download all the files listed in the message from the storage server to local storage.
+    for file_name, file_transfer_token in file_transfer_tokens.items():
+        try:
+            file_transfer_token.sig = None
+            cluster_response = send_request_to_cluster_storage(
+                prepare_storage_request_headers(file_transfer_token),
+                settings.STORAGE_CLUSTER_ADDRESS + CLUSTER_DOWNLOAD_PATH + file_transfer_token.files[0]['path'],
+                method='get',
+            )
+            store_file_from_response_in_chunks(
+                cluster_response,
+                os.path.join(settings.VERIFIER_STORAGE_PATH, file_name)
+            )
+
+        except (OSError, HTTPError) as exception:
+            verification_result.delay(
+                subtask_id,
+                VerificationResult.ERROR,
```

Comment:
Please log something if you're silencing an error.

Reply:
:+1: done